api/core/v1alpha1/etcd.go

```diff
@@ -188,6 +188,12 @@ type BackupSpec struct {
 	// EnableProfiling defines if profiling should be enabled for the etcd-backup-restore-sidecar
 	// +optional
 	EnableProfiling *bool `json:"enableProfiling,omitempty"`
+	// EnableMemberLeaseRenewal defines if member lease renewal should be enabled for the etcd-backup-restore-sidecar
+	// +optional
+	EnableMemberLeaseRenewal *bool `json:"enableMemberLeaseRenewal,omitempty"`
```

Comment:
Currently, we have not exposed the `--enable-{member,snapshot}-lease-renewal`  flags via API because we assume that `etcd-druid` will always operate within Kubernetes environments. From an open-source perspective and consumption of etcd-druid outside gardener, it is not advisable to allow users to set these flags to `false`, as doing so would make `etcd-druid` unable to scale up the etcd cluster. 
These flags should only be set to `false` in non-HA conditions where there is no intention to transition to HA in the future.

Reply:
`gardenadm` (as bring-up tool for autonomous shoot clusters) is also just a user of `etcd-druid` and its `Etcd` API. It requires these flags to be set to `false` on `etcd-backup-restore`. Hence, we need to make this configurable somehow, be it via `spec` or via annotation as proposed by https://github.com/gardener/etcd-druid/pull/1087#pullrequestreview-2827028689